book/src/http_beacon.md

```diff
@@ -233,6 +236,105 @@ and its tree hash root.
 
 _Truncated for brevity._
 
+## `/beacon/state/genesis`
+
+Request that the node return a beacon chain state at genesis (slot 0).
+
+### HTTP Specification
+
+| Property | Specification |
+| --- |--- |
+Path | `/beacon/state/genesis`
+Method | GET
+JSON Encoding | Object
+Query Parameters | None
+Typical Responses | 200, 404
+
+### Parameters
```

Comment:
Maybe we should say "No Parameters" or something like that? I'm down with whatever is consistent with what we already have.

Reply:
I guess simply removing the `Parameters` heading when `Query Parameters` is the best choice and consistent with what we already have.